KomaMRICore/test/test_files/Utils.jl

```diff
@@ -0,0 +1,72 @@
+using HDF5
+
+function signal_jemris()
+    path = @__DIR__
+    sig = h5open(joinpath(path, "jemris_signals_epi_sphere_cs.h5"))["/signal/channels/00"]
+    sig = sig[1,:] + 1im*sig[2,:]
+    sig = sig[:]
+    return sig
+end
+
+function phantom_sphere()
+    path = @__DIR__
+    fid = h5open(joinpath(path, "phantom_sphere.h5"), "r")
+    x, y, z, ρ = fid["x"][:], fid["y"][:], fid["z"][:], fid["ρ"][:]
+    T1, T2, T2s, Δw = fid["T1"][:], fid["T2"][:], fid["T2s"][:], fid["Δw"][:]
+    return Phantom(; name="sphere", x, y, z, ρ, T1, T2, T2s, Δw)
+end
+
+function seq_epi_100x100_TE100_FOV230()
+
+    # Gyromagnetic constant
+    γ_jemris = 2π*γ*1e-9            # γ in [Hz/T]. γ_jemris in [rad/us/mT]
```

Comment:
```suggestion
    γ_rad_us_mT = 2π*γ*1e-9            # γ in [Hz/T]. γ_jemris in [rad/us/mT]
```

Reply:
Addressed in https://github.com/cncastillo/KomaMRI.jl/pull/240/commits/c6ec91b5c89bc94b578a6af803e90f3709dcb5cb